docker/runtime/README.md

```diff
@@ -0,0 +1,137 @@
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## How to build a docker image
+
+### Preparation 
+
+#### Download and copy the Doris code repo
+
+```shell
+$ cd /to/your/workspace/
+$ git clone https://github.com/apache/doris.git
+$ cd doris/docker
+$ cp -r runtime/ /to/your/workspace/
+```
+
+After preparation, your workspace should like this:
+
+```
+.
+├── runtime
+│   ├── README.md
+│   ├── all-in-one
+│   ├── base-image
+│   ├── be
+│   ├── broker
+│   ├── docker-compose-demo
+│   ├── doris-compose
+│   ├── fe
+│   ├── ms
+```
+
+#### Download, unzip and copy the antidote package
+
+Here we take the build of x64 (avx2) and arm64 platforms of 2.1.5 version as an example.
+
+1. Download binary
+
+Go to the doris [official website](https://doris.apache.org/download) to download the binary package you need, and pay attention to selecting the doris version and architecture type you need.
+  
```

Comment:
Here we take the build of x64 (avx2) and arm64 platforms of 2.1.5 version as an example.